### PR TITLE
Fail the build on snark profiler failure

### PR DESCRIPTION
**Pull request by dkijania**
Post mortem after finding out test snark profiler test passed in CI while we encounter some Errors in snark profiler output. 

For example silent fail:

https://buildkite.com/o-1-labs-2/mina-end-to-end-nightlies/builds/2629#01924565-5234-403a-a08b-5c0bac6dd200

The solution is to take a look at output too and if it contains invalid content, fails the build.

### CODE REVIEW

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Review by SanabriaRusso — APPROVED** [review]

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

File: scripts/snark_transaction_profiler.py [hunk 1]
```diff
@@ -63,13 +58,11 @@ def parse_stats (output) :
...
     print(f'running snark transaction profiler: {args}')
     (process_exit_code,output) = subprocess.getstatusoutput(args)
+    
     stats = parse_stats (output)
     #TODO: add code to check against some threshold
     print(stats)
     
-
     if not process_exit_code == 0:
         print('non-zero exit code from program, failing build')
```
Comment (deepthiskumar) [inline scripts/snark_transaction_profiler.py:67]: print the exit code as well, for debugging?

**Review by deepthiskumar — APPROVED** [review]

**Comment by dkijania** [comment]
!ci-build-me

**Comment by dkijania** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!ci-build-me

**Comment by deepthiskumar** [comment]
!approved-for-mainnet

**Pull request merged**